exporter/splunkhecexporter/factory.go

```diff
@@ -57,6 +58,23 @@ func NewFactory() exporter.Factory {
 		exporter.WithLogs(createLogsExporter, metadata.LogsStability))
 }
 
+func getExporterOptions(cfg *Config, startFunc component.StartFunc, shutdownFunc component.ShutdownFunc) []exporterhelper.Option {
+	options := []exporterhelper.Option{
+		// explicitly disable since we rely on http.Client timeout logic.
+		exporterhelper.WithTimeout(exporterhelper.TimeoutSettings{Timeout: 0}),
+		exporterhelper.WithRetry(cfg.BackOffConfig),
+		exporterhelper.WithQueue(cfg.QueueSettings),
+		exporterhelper.WithStart(startFunc),
+		exporterhelper.WithShutdown(shutdownFunc),
+	}
+
+	if cfg.BatcherConfig.Enabled {
```

Comment:
This condition is not supposed to be needed. Can be dropped after https://github.com/open-telemetry/opentelemetry-collector/pull/10076 is merged

Reply:
approved that change.